packages/frontend/src/components/common/Dashboard/DashboardHeader.tsx

```diff
@@ -267,7 +273,32 @@ const DashboardHeader = ({
                 <PageActionsContainer>
                     {userCanExportData && <DashboardRefreshButton />}
 
-                    {!!userCanManageDashboard && (
+                    {!isEditMode && document.fullscreenEnabled && (
```

Comment:
can we use 
```suggestion
                    {!isEditMode && isFullScreen && (
``` 
?

Reply:
The check on `document.fullscreenEnabled` is used to hide the button if fullscreen isn't available in this user agent, despite the super confusing name 😵‍💫